### ISSUE TITLE
ProgressStepper - Popovers should be interactive via keyboard

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-react#7200]
**Is this a new component or an extension of an existing one?**
Existing

**Describe the feature**
For the [Progress Stepper with Popover](https://www.patternfly.org/v4/components/progress-stepper#with-help-popover) example, the popovers can only be opened via mouse click or with Voice Over's VO + Space keyboard interaction (which acts as a click event I believe). Using just the keyboard, the popover cannot be opened with the Enter or Space keys like a button normally should.

The issue might be due to the fact that when a Popover is rendered for a ProgressStep it renders a `<span>` with a role attribute and value of "button" rather than a native `<button>` element. Using a `<button>` element might require a similar fix to be made in Core, however.

**Are there visuals for this feature? If applicable, please include examples for each state and for varying widths**
N/A

**Any other information?**
N/A
